serpentTools/parsers/xsplot.py

```diff
@@ -120,4 +121,7 @@ def _postcheck(self):
                   "The incomplete XS is {}".format(xs.name))
 
         # check energy grid found
-        assert 'egrid' in self.metadata.keys()
+        try:
+            assert 'egrid' in self.metadata.keys()
+        except AssertionError as e:
+            e.args += 'No energy grid found in xsplot parser.'
```

Comment:
Will this raise the AssertionError or simply append to `args`?

Reply:
This prints additonal info with the AssertionError. Reference: https://stackoverflow.com/questions/3807694/how-to-change-the-message-in-a-python-assertionerror